entrypoint.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+# Change ownership of /palworld to steam user
+chown -R steam:steam /palworld
+# Run servermanager.sh as steam user
+su -c "/servermanager.sh" - steam
```

Comment:
`su -c "/servermanager.sh" - steam` this breaks because it doesn't load the correct environment variables

change to this

`su -c "/servermanager.sh" steam`

The - (dash) parameter in the `su` command is used to provide an environment similar to what the user would expect had the user logged in directly. This means it initiates a login shell and sources the profile files, which changes the environment variables, because they are being overwritten in the user's profile files.

This fix will run the commands in a non-login shell, which does not source the profile files and preserves the correct environment variables.